steward/src/config.rs

```diff
@@ -30,19 +30,18 @@ lazy_static! {
 #[derive(Clone, Debug, Deserialize, Serialize)]
 #[serde(default, deny_unknown_fields)]
 pub struct StewardConfig {
-    pub keystore: String,
-    pub server: ServerSection,
-    pub gravity: GravitySection,
-    pub ethereum: EthereumSection,
-    pub cosmos: CosmosSection,
-    pub metrics: MetricsSection,
     pub cellars: Vec<CellarConfig>,
+    pub cosmos: CosmosSection,
```

Comment:
I appreciate the idea behind alphabetizing these, but I think having the config divided up by sections grouped by their relevancy is probably more useful to someone reading the config, e.g. global config, an orchestrator/relayer grouping, a steward grouping, etc.

Reply:
I was going to make an issue for refactoring config into steward/orchestrator tables but thought it would be left for later. None of the tables are really clear about what parts of steward used them. Should I change the ordering again or make an issue?

Reply:
I think it's fine to leave it as you have it here for now.